tools/nixpkgs-hammer

```diff
@@ -224,4 +224,9 @@ if __name__ == '__main__':
 
     args = parser.parse_args()
 
-    main(args)
+    try:
```

Comment:
I was thinking about moving this around the `all_messages` assignment.

Reply:
How about I remove this commit from this PR, and you can work on a different solution for improving the error messages when things crash?

Reply:
Sounds good to me.

Reply:
Cool. I filed #18 and #19 for this.